js/os.ts

```diff
@@ -37,11 +37,14 @@ export function codeFetch(
   fbs.Base.addMsgType(builder, fbs.Any.CodeFetch);
   builder.finish(fbs.Base.endBase(builder));
   const resBuf = libdeno.send(builder.asUint8Array());
+  assert(resBuf != null);
   // Process CodeFetchRes
-  const bb = new flatbuffers.ByteBuffer(new Uint8Array(resBuf));
+  // TypeScript does not track `assert` from a CFA perspective, therefore not
+  // null assertion `!`
+  const bb = new flatbuffers.ByteBuffer(new Uint8Array(resBuf!));
```

Comment:
These null assert checks are redundant when using the bang operator. Suggest removing the assert.